services/offline/src/lib/__tests__/online-status.test.js

```diff
@@ -39,7 +39,7 @@ describe('state changes in response to browser "online" and "offline" events', (
         })
 
         // Wait for debounce
-        await waitForNextUpdate({ timeout: 1000 })
+        await waitForNextUpdate({ timeout: 1010 })
```

Comment:
Stopped working when bumping package versions; small change fixes it

Reply:
It's a shame we have to pass a timeout to `waitForNextUpdate` instead of mocking the `setTimeout`s used by `lodash.debounce` as each such test increases the time it takes to run the test suite by ~1s.

I tried updating this test to use `jest.useFakeTimers` with `jest.runOnlyPendingTimers`/`jest.advanceTimersByTime(1000)` but couldn't get it to work. According to some of the comments on https://github.com/facebook/jest/issues/3465, once solution would be to mock `lodash.debounce` but I feel like that's going a bit too far.

Reply:
Yeah, I also tried the fake timers to no avail :(

But maybe bumping jest would help? It looks like the released a fix: https://github.com/facebook/jest/issues/3465#issuecomment-623393230

Reply:
I tried with `jest.useFakeTimers('modern')` and `yarn.lock` says we're using `jest@26.6.3` :shrug:

Reply:
Interesting - it seems to be running jest@24.9.0 for me